test/onnx/test_pytorch_onnx_shape_inference.py

```diff
@@ -268,6 +271,60 @@ def test_resize_after_concat(self):
         )
         self.run_test(g, resize.node(), expect_tensor("Float", shape=(4, 32, 128, 128)))
 
+    def test_autograd_layernorm_shape(self):
+        def symbolic_pythonop(ctx: torch.onnx.SymbolicContext, g, *args, **kwargs):
+            return g.op("com.microsoft::PythonOp")
+
+        torch.onnx.register_custom_op_symbolic("prim::PythonOp", symbolic_pythonop, 1)
+        self.addCleanup(torch.onnx.unregister_custom_op_symbolic, "prim::PythonOp", 1)
+
+        # necessay parameters for transformer embeddings
+        hidden_size, max_position_embeddings, batch_size = (
+            48,
+            32,
+            2,
+        )
+
+        # wrap nn.layernorm into autograd.func repro the same
+        # issue as using apex fusedlayernorm
```

Comment:
Prefer a more detailed description of the “issue”, possibly a link to the issue itself so readers know why the test exists and when it no longer applies

Reply:
I found a paradox that I am not actually using Apex FusedLayerNorm at any point, and keep mention it, as I wanted the people having same issue to find this. Iguess the better way would be that I raise an issue about how this is discovered, and put Apex FusedLayerNorm there for people having the same issue to find it.